include/LeMonADE/updater/moves/MoveAddBccMonomer.h

```diff
@@ -115,8 +93,8 @@ void MoveAddBccMonomer::apply(IngredientsType& ing)
 {
   //first add the new monomer at the desired position. this is because
   //some features may want to do things with it
-  ing.modifyMolecules().addMonomer(position.getX(),position.getY(),position.getZ());
-  particleIndex=ing.getMolecules().size()-1;
+  ing.modifyMolecules().addMonomer(this->getPosition().getX(),this->getPosition().getY(),this->getPosition().getZ());
+  this->setParticleIndex(ing.getMolecules().size()-1);
   //now apply it to the features so that the features can make alterations,
   //for example set the attribute tag, if the FeatureAttributes is used
   ing.applyMove(ing,*this);		
```

Comment:
At the bottom-line there's written `#endif //LEMONADE_UPDATER_MOVES_MOVEADDSCMONOMER_H`. It should be read 
`#endif //LEMONADE_UPDATER_MOVES_MOVEADDBCCMONOMER_H` 
or 
`#endif //LEMONADE_UPDATER_MOVES_MOVEADDMONOMERBCC_H`